.github/workflows/android.yml

```diff
@@ -0,0 +1,48 @@
+name: Build APK
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
```

Comment:
we don't need to run the action on pull request

Reply:
Why not

Reply:
because pull request merges are considered pushes, we don't need a separate build just for the PR itself

Reply:
Well having it on pull request is good so you can make sure prs actually build before merging